examples/rbac/verify.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+export NAME=rbac
+export PORT_PROXY="${RBAC_PORT_PROXY:-10000}"
```

Comment:
the reason for doing this is that we want all of the examples to use different ports (this will allow the examples to be verified simultaneously)

my very hokey system for doing it involves the first letter and its place in the alphabet + an additional integer (prefixed by 1)

in this case lets use:

`11810` and `11811`

Reply:
Thanks. I have been puzzled by these ports in examples. I will update them.